### PR TITLE
Added restricted form field, cleaned example application

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: chamber/forms/fields.py [hunk 1]
```diff
@@ -38,3 +43,20 @@ def __init__(self, *args, **kwargs):
         if 'widget' not in kwargs:
             kwargs['widget'] = PriceNumberInput(currency)
         super().__init__(*args, **kwargs)
+
+
+class RestrictedFileField(forms.FileField):
```
Comment (matllubos) [inline chamber/forms/fields.py]: Like model RestrictedFileField

File: chamber/forms/validators.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import mimetypes
```
Comment (matllubos) [inline chamber/forms/validators.py]: only move to new file

File: example/requirements.txt [hunk 3]
```diff
@@ -1,10 +1,11 @@
-Django==1.11
+Django==2.0
```
Comment (matllubos) [inline example/requirements.txt]: 1.11 has security issue

File: example/dj/apps/test_chamber/tests/models/fields.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 from chamber.models.fields import generate_random_upload_path
 from chamber.shortcuts import change_and_save
 
-from germanium.anotations import data_provider
+from germanium.decorators import data_provider
```
Comment (matllubos) [inline example/dj/apps/test_chamber/tests/models/fields.py]: used new germanium

File: docs/forms.rst [hunk 5]
```diff
@@ -12,3 +12,10 @@ A widget for safe rendering of readonly form values.
 .. class:: chamber.forms.fields.PriceField
 
 ``django.forms.NumberInput`` with ``currency`` as a placeholder.
+
+.. class:: chamber.forms.fields.RestictedFileField
+
+``django.forms.FileField`` where you can set ``allowd_content_types`` amd ``max_upload_size``. File type is validated by file extension and content::
```
Comment (Formulka) [inline docs/forms.rst]: allowd_content_types -> allowed_content_types

**Pull request merged**